src/main/kotlin/org/eclipse/uprotocol/cloudevent/validate/CloudEventValidator.kt

```diff
@@ -241,21 +231,18 @@ abstract class CloudEventValidator {
         }
 
         override fun validateSink(cloudEvent: CloudEvent): ValidationResult {
-            val maybeSink: Optional<String> = UCloudEvent.getSink(cloudEvent)
-            if (maybeSink.isEmpty) {
-                return ValidationResult.failure("Invalid CloudEvent sink. Response CloudEvent sink must be uri the destination of the response.")
-            } else {
-                val sink: String = maybeSink.get()
-                val checkSink: ValidationResult = validateRpcTopicUri(sink)
-                if (checkSink.isFailure()) {
-                    return ValidationResult.failure(
-                        String.format(
-                            "Invalid RPC Response CloudEvent sink [%s]. %s",
-                            sink,
-                            checkSink.getMessage()
-                        )
+            val sink: String = UCloudEvent.getSink(cloudEvent)
+                ?: return ValidationResult.failure("Invalid CloudEvent sink. Response CloudEvent sink must be uri the destination of the response.")
+            val checkSink: ValidationResult = validateRpcTopicUri(sink)
+            if (checkSink.isFailure()) {
+                return ValidationResult.failure(
+                    String.format(
+                        "Invalid RPC Response CloudEvent sink [%s]. %s",
+                        sink,
+                        checkSink.getMessage()
```

Comment:
Sounds like all those validateSink are very similar. Should we refactor with a private method here?